iml-gui/crate/src/page/filesystem.rs

```diff
@@ -421,8 +421,8 @@ fn targets(
                 t::thead_view(vec![
                     t::th_left(plain!["Name"]).merge_attrs(class![C.w_32]),
                     t::th_left(plain!["Volume"]),
-                    t::th_left(plain!["Primary Server"]).merge_attrs(class![C.w_48]),
-                    t::th_left(plain!["Failover Server"]).merge_attrs(class![C.w_48]),
+                    t::th_left(plain!["Primary Server"]).merge_attrs(class![C.w_48, C.hidden, C.md__table_cell]),
+                    t::th_left(plain!["Failover Server"]).merge_attrs(class![C.w_48, C.hidden, C.md__table_cell]),
                     t::th_left(plain!["Started on"]).merge_attrs(class![C.w_48]),
```

Comment:
I was thinking how we could put all these servers in one cell. E. g. show only "started on" and mark it somehow whether it's the primary or a failover. Maybe show all servers in a popup.

Reply:
Yeah it's a good thought. Once things slow down perhaps we could look through the mocks a bit and make changes where necessary.